honeybee_radiance/modifier/material/bsdf.py

```diff
@@ -74,31 +72,20 @@ class BSDF(Material):
         * is_modifier
         * is_material
     """
-    __slots__ = ('_bsdf_file', '_up_orientation', '_thickness', '_function_file',
-                 '_transform', '_angle_basis', '_front_diffuse_reflectance',
-                 '_back_diffuse_reflectance', '_diffuse_transmittance')
+
+    __slots__ = ('_thickness')
```

Comment:
A tuple with a single item and no ',' will be considered a string!

```suggestion
    __slots__ = ('_thickness',)
```